doc/shelley-genesis.md

```diff
@@ -1084,14 +1092,14 @@ the `genesis.json`.
 There is a command to query the UTxO and return all the UTxOs at a given
 address.
 ```
-$ cardano-cli shelley query filtered-utxo
-Usage: cardano-cli shelley query filtered-utxo --address ADDRESS
-                                               (--mainnet | --testnet-magic INT)
-                                               [--out-file FILE]
-  Get the node's current UTxO filtered by address
+$ cardano-cli shelley query utxo
+Usage: cardano-cli shelley query utxo [--address ADDRESS]
```

Comment:
In this line address in now in square brackets which suggests that it is optional param [--address ADDRESS] So this should probably be changed.

Reply:
What should be changed exactly? Yes, the address flag is optional. You can indeed ask for the whole UTxO.

Reply:
My bad, I though that some address should be specified, it's an old PR and I forgot about it. I will close it.